app/ui/browser/index.jsx

```diff
@@ -73,7 +73,6 @@ store.dispatch(actions.createTab());
 // that if an error occurs while we connect, we at least have some UI in place.
 ReactDOM.render(chrome, container);
 
-userAgentClient.connect().then(() => onWindowReady(true));
```

Comment:
The `onWindowReady` is still inside the first diff handler -- just tested in production mode, and we still have syncing bookmarks!